### PR TITLE
Add gitql package

### PR DESCRIPTION
**Pull request by AmrDeveloper**
Closes ##5095

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/gitql.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "description": "SQL like lanuage to perform queries on .git files",
+    "homepage": "https://github.com/amrdeveloper/gql",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/AmrDeveloper/GQL/releases/download/0.6.0/gql-x86_64-windows.exe",
+            "hash": "0b7dc2b055204d12eae9f8e29f2755d7df1a63f527af10340a489b42328b6e3f"
+        }
+    },
+    "pre_install": "Rename-Item \"$dir\\gql-x86_64-windows.exe\" \"$dir\\gitql.exe\"",
```
Comment (aliesbelik) [inline bucket/gitql.json:12]:
```suggestion
```

Unnecessary if `#/` operator used to rename.

File: bucket/gitql.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.6.0",
+    "description": "SQL like lanuage to perform queries on .git files",
+    "homepage": "https://github.com/amrdeveloper/gql",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/AmrDeveloper/GQL/releases/download/0.6.0/gql-x86_64-windows.exe",
+            "hash": "0b7dc2b055204d12eae9f8e29f2755d7df1a63f527af10340a489b42328b6e3f"
+        }
```
Comment (aliesbelik) [inline bucket/gitql.json:10]:
```suggestion
        },
        "32bit": {
            "url": "https://github.com/AmrDeveloper/GQL/releases/download/0.6.0/gql-x86-windows.exe#/gitql.exe",
            "hash": "cdd3d5055e6246b7807a2d97a1022b2c805fe81a0f795a7b5f4411b5b70ed359"
        }
```

Please add 32bit version as well if supported.

File: bucket/gitql.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "pre_install": "Rename-Item \"$dir\\gql-x86_64-windows.exe\" \"$dir\\gitql.exe\"",
+    "bin": "gitql.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/AmrDeveloper/GQL/releases/download/$version/gql-x86_64-windows.exe"
+            }
```
Comment (aliesbelik) [inline bucket/gitql.json:19]:
```suggestion
            },
            "32bit": {
                "url": "https://github.com/AmrDeveloper/GQL/releases/download/$version/gql-x86-windows.exe#/gitql.exe"
            }
```

For autoupdate as well.

File: bucket/gitql.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        }
```
Comment (aliesbelik) [inline bucket/gitql.json:20]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/$basename.sha256"
        }
```

Link to checksums files if available.

File: bucket/gitql.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.6.0",
+    "description": "SQL like lanuage to perform queries on .git files",
+    "homepage": "https://github.com/amrdeveloper/gql",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/AmrDeveloper/GQL/releases/download/0.6.0/gql-x86_64-windows.exe",
```
[the remark this answers is not in the document]
Comment (AmrDeveloper) [reply bucket/gitql.json]: Done, Thanks for suggestion @aliesbelik

File: bucket/gitql.json (hunk 1 above)
Comment (AmrDeveloper) [reply bucket/gitql.json]: Done, Thanks for suggestion @aliesbelik

File: bucket/gitql.json (hunk 2 above)
Comment (AmrDeveloper) [reply bucket/gitql.json:14]: Done

File: bucket/gitql.json (hunk 3 above)
Comment (AmrDeveloper) [reply bucket/gitql.json:28]: Done

File: bucket/gitql.json (hunk 4 above)
Comment (AmrDeveloper) [reply bucket/gitql.json:29]: CI: Property 'hash' has not been defined and the schema does not allow additional properties

Comment (aliesbelik) [reply bucket/gitql.json:29]: > CI: Property 'hash' has not been defined and the schema does not allow additional properties

Sorry, my fault while inserting.
`hash` section should be under `autoupdate`, on same level as `architecture`.

Comment (AmrDeveloper) [reply bucket/gitql.json:29]: No problem, i fixed it :D

Thanks for suggesting

Comment (AmrDeveloper) [reply bucket/gitql.json:29]: Done it passed @aliesbelik

**Review by aliesbelik — CHANGES_REQUESTED** [review]

**Pull request merged**